src/font/text.cpp

```diff
@@ -73,6 +98,7 @@ pango_text::pango_text()
 	pango_layout_set_ellipsize(layout_.get(), ellipse_mode_);
 	pango_layout_set_alignment(layout_.get(), alignment_);
 	pango_layout_set_wrap(layout_.get(), PANGO_WRAP_WORD_CHAR);
+
```

Comment:
Could you add a comment here that we want to phase this out in favor of a global line height attribute?